incubator/burrow/README.md

```diff
@@ -2,6 +2,8 @@
 
 [burrow](https://github.com/hyperledger/burrow) is a permissioned Ethereum smart-contract blockchain node. It executes Ethereum smart contract code on a permissioned virtual machine. Burrow provides transaction finality and high transaction throughput on a proof-of-stake Tendermint consensus engine.
 
+**Note** this repository has been [moved to stable](../../stable/burrow).
```

Comment:
Please mention explicitly that the chart (not the repo) has been deprecated. Here's an example:
https://github.com/kubernetes/charts/blob/5e8a4639e05c3e6f4c9765881b0b26f97e12e098/incubator/oauth-proxy/README.md#L5

Reply:
OK. Changed it now. It would be good to document this process with a bit more particularity to save the back and forth.